.travis.yml

```diff
@@ -33,7 +33,9 @@ env:
     # - RUN_BACKEND_TESTS=true REPORT_BACKEND_COVERAGE=true
     - RUN_LINT=true
     - RUN_FRONTEND_TESTS=true
-    - RUN_PERFORMANCE_TESTS=true
+    # TODO(sll): Reinstate this when the flakiness issues have been fixed.
+    # Currently, on Travis-CI, the test often fails to terminate.
+    # - RUN_PERFORMANCE_TESTS=true
```

Comment:
Prefer not commenting it out; can you just remove it? The tracking issue provides context and this PR provides history when we need to roll-forward the reinstating of performance tests on Travis.